crates/cast/src/starknet_commands/account/add.rs

```diff
@@ -0,0 +1,82 @@
+use crate::starknet_commands::account::{prepare_account_json, write_account_to_accounts_file};
+use anyhow::{ensure, Result};
+use camino::Utf8PathBuf;
+use cast::helpers::response_structs::AccountAddResponse;
+use cast::helpers::scarb_utils::CastConfig;
+use clap::Args;
+use starknet::core::types::FieldElement;
+use starknet::signers::SigningKey;
+
+#[derive(Args, Debug)]
+#[command(about = "Add an account to the accounts file")]
+pub struct Add {
+    /// Address of the account
+    #[clap(short, long)]
+    pub address: FieldElement,
+
+    /// Class hash of the account
+    #[clap(short, long)]
+    pub class_hash: Option<FieldElement>,
+
+    /// Account deployment status
+    #[clap(short, long)]
+    pub deployed: bool,
```

Comment:
I'm worried this might be too vague for users. I don't think we should require this to be passed; I suppose we could try to fetch the account to see if it was deployed and include the info by ourselves, or alternatively just don't include this and assume it was deployed.

Reply:
I'll think some alternative